src/eei.cpp

```diff
@@ -468,9 +477,9 @@ namespace HeraVM {
       context->fn_table->call(&call_result, context, &call_message);
 
       if (call_result.output_data) {
-        vector<uint8_t> result(call_result.output_data, call_result.output_data + call_result.output_size);
-        result.resize(resultLength);
-        storeMemory(result, 0, resultOffset, resultLength);
+        lastReturnData.assign(call_result.output_data, call_result.output_data + call_result.output_size);
```

Comment:
Is call not supposed to store output data at resultOffset anyway?

Reply:
There is no point doing that with returndata anymore. See https://github.com/ewasm/design/issues/70

Reply:
But you are correct, perhaps I should separate these changes into two: supporting returndata and removing that option.

Reply:
Done, that will be a different PR.